docs/root/intro/arch_overview/service_discovery.rst

```diff
@@ -84,28 +84,19 @@ preferred service discovery mechanism for a few reasons:
   load balancing weight, canary status, zone, etc. These additional attributes are used globally
   by the Envoy mesh during load balancing, statistic gathering, etc.
 
-Generally active health checking is used in conjunction with the eventually consistent service
-discovery service data to making load balancing and routing decisions. This is discussed further in
-the following section.
-
-.. _arch_overview_service_discovery_types_sds:
-
-Service discovery service (v1 SDS)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Envoy project provides reference gRPC implementations of EDS and
+:ref:`other discovery services <arch_overview_dynamic_config>`
+in both `Java <https://github.com/envoyproxy/java-control-plane>`_
+and `Go <https://github.com/envoyproxy/go-control-plane>`_.
 
-*(deprecated)* This discovery service was replaced to endpoint discovery service in :ref:`v2 xDS
-API<config_overview_v2>`.
-
-The *service discovery service* is a generic :ref:`REST based API <config_cluster_manager_sds_api>`
-used by Envoy to fetch cluster members. Lyft provides a reference implementation via the Python
+Lyft provides a reference REST implementation via the Python
```

Comment:
The Lyft version here doesn't actually support v2 and won't. Let's just delete this entirely.

Reply:
Ah! Sorry, I meant to delete that, but forgot to after adding the other stuff.